frontend/src/pages/admin/categories/index.tsx

```diff
@@ -29,12 +30,29 @@ const Categories: NextPage = (): JSX.Element => {
   const pageCount = Math.ceil(categories?.data?.length / cagoryPerPage)
 
   const changePage = ({ selected }: { selected: number }): void => setPageNumber(selected)
+  
+  const handleDelete = async (id: string): Promise<void> => {
+    let result = confirm('Are you sure you want to delete?')
```

Comment:
you can use const here since you are not updating the value
```suggestion
    const result = confirm('Are you sure you want to delete?')
```